rust/common/error/macros.rs

```diff
@@ -76,6 +89,16 @@ macro_rules! error_messages {
             }
         }
 
+        impl std::fmt::Debug for $name {
+            fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+                let mut debug_struct = f.debug_struct(&self.name());
```

Comment:
This would call the struct e.g. `MultiLine` instead of the more appropriate `TestError::Multiline`

Reply:
We now use the full type name, as per https://doc.rust-lang.org/std/any/fn.type_name.html